rust/kcl-lib/src/execution/exec_ast.rs

```diff
@@ -1203,10 +1211,12 @@ impl Node<BinaryExpression> {
 
     fn warn_on_unknown(&self, ty: &NumericType, verb: &str, exec_state: &mut ExecState) {
         if ty == &NumericType::Unknown {
-            // TODO suggest how to fix this
+            let sr = self.as_source_range();
+            exec_state.clear_units_warnings(&sr);
+            // The part of the message "unknown or incompatible units" is significant. If you change it, update `clear_unit_warnings`.
```

Comment:
If the string is significant, can we make it a `const` and reference it in all the places that care? Or better yet, a new field that's an enum?